### PR TITLE
new HTML added

### PR DESCRIPTION
**Pull request by colindamelio**
- added our homepage HTML! 🔥

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,15 @@
 	<meta charset="UTF-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Document</title>
+	<title>We rock!</title>
```
Comment (darshanasen) [inline index.html]:
```suggestion
	<title>Sonic the Hedgehog - homepage</title>
```

File: index.html [hunk 2]
```diff
@@ -4,9 +4,15 @@
...
 </head>
 <body>
+
+	<h1>Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia, voluptas laborum consequuntur sunt temporibus ab adipisci consectetur molestias odio quae voluptate ea dolore nostrum omnis reiciendis aliquam asperiores blanditiis! Laudantium.</h1>
```
Comment (darshanasen) [inline index.html]: Really love the usage of a semantic heading - perhaps consider putting this within a `header` element.

File: index.html [hunk 3]
```diff
@@ -4,9 +4,15 @@
...
+
+	<h2>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quaerat, repellendus molestias suscipit quasi, incidunt eaque amet hic odio quae, animi veniam! Laborum, nihil fuga nobis ut exercitationem cupiditate quos ea?</h2>
 	
+	<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus sapiente velit doloremque pariatur harum ullam consectetur corporis quidem voluptatum. Asperiores, enim dolore natus expedita dicta facilis quo corporis labore. Repudiandae.</p>
```
Comment (darshanasen) [inline index.html]:
```suggestion
	<h3>Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus sapiente velit doloremque pariatur harum ullam consectetur corporis quidem voluptatum. Asperiores, enim dolore natus expedita dicta facilis quo corporis labore. Repudiandae.</h3>
```

**Review by darshanasen — CHANGES_REQUESTED** [review]
Really great initial structure. I've left a few comments for your consideration about better semantic HTML structure.

File: index.html (hunk 2 above)
Comment (colindamelio) [reply index.html]: great tip, this is fixed.

**Review by darshanasen — APPROVED** [review]
AMAZING updates! Sonic would be so proud 🏅

**Pull request merged**